src/client/api/init.c

```diff
@@ -123,6 +123,60 @@ const struct daos_task_api dc_funcs[] = {
 	{dc_kv_put, sizeof(daos_kv_put_t)},
 	{dc_kv_remove, sizeof(daos_kv_remove_t)},
 	{dc_kv_list, sizeof(daos_kv_list_t)},
+	{dc_reserved, 0},
+	{dc_reserved, 0},
+
+	/** More Pool */
+	{dc_pool_filter_cont, sizeof(daos_pool_filter_cont_t)},
+
+	/** Reserved for More Pool, Container, Transaction, Object, Array, Key-Value Store */
```

Comment:
as discussed, can just removed this dc_reserved for now.

Reply:
done in latest version of the patch